R/fusion_driver.R

```diff
@@ -7,10 +7,13 @@
 #' @param annotated Boolean if annotated
 #' @param geneListReferenceDataTab A dataframe with column 1 as GeneName 2 source file 3 type; collapse to summarize type
 #' @param fusionReferenceDataTab A dataframe with column 1 as FusionName 2 source file 3 type; collapse to summarize type
+#' @param checkDomainStatus Logical value to check if domain status in fused gene for given domansToCheck, default to FALSE
```

Comment:
```suggestion
#' @param checkDomainStatus Logical value to check if domain status in fused gene for given domainsToCheck, default to TRUE
```

also think this should be default -== T